msteams-platform/bots/how-to/authentication/auth-aad-sso-bots.md

```diff
@@ -217,6 +217,49 @@ The response with the token is sent through an invoke activity with the same sch
 
 The `turnContext.activity.value` is of type [TokenExchangeInvokeRequest](/dotnet/api/microsoft.bot.schema.tokenexchangeinvokerequest?view=botbuilder-dotnet-stable&preserve-view=true) and contains the token that can be further used by your bot. You must store the tokens for performance reasons and refresh them.
 
+### Token exchange failure
+
+In case of token exchange failure, use the following code:
+
+```json
+{​​ "status": "<response code>", "body": {​​ "id":"<unique Id>", "connectionName": "<connection Name on the bot (from the OAuth Card)>", "failureDetail": "<failure reason if status code is not 200, null otherwise>" }​​ }​​
+```
```

Comment:
Shouldn't this be appropriately indented? Will the json code work in this format?